docs/maya/simple_setup.md

```diff
@@ -31,21 +31,21 @@ To create the AdnSkin deformer with some initial customisation, double-click the
 
 Once the AdnSkin deformer is properly created it is possible now to paint its weights to correctly setup the deformer properties. To do so, select the simulated mesh and press the ![paint tool](images/adn_paint_tool.png){style="width:4%"} shelf button or go to AdonisFX Menu > *Paint Tool*.
 
-Start by painting *Soft Constraints* by selecting the option from the attribute enumerator. Flood this weight to a low value of 0.2 to have a uniform distribution of soft constraints. This will help the skin to follow the reference mesh.
+Start by painting *Soft Constraints* by selecting the option from the attribute enumerator. Flood this weight to a low value of 0.2 to have a uniform distribution of soft constraints. This will help the skin to follow the target mesh.
```

Comment:
I think we should be more verbose in saying that it will follow the target meshes in plural. What do you think?

Reply:
We should review that for all doc pages. We don't want to give the user the impression now that there will only be 1 target at most

Reply:
Or at least be more specific that it's the target that was found using the closest point

Reply:
I get you, but I'm not totally sure here because the specific example for AdnSkin in "A Simple Setup" page is using only one target mesh, that is why I was not using the plural. It is true that we have to make it clear to the user that it is possible to use more than one target mesh, although I think that is clear by reading the skin page. Let me know what you think

Reply:
I see. Then let's leave it specifically for the use case of using one single target